draft-ietf-rtcweb-jsep.xml

```diff
@@ -2677,6 +2671,13 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
               <t>An "a=dtls-id" line, as specified in
               <xref target="I-D.ietf-mmusic-dtls-sdp"/> Section 5.3.</t>
 
+              <t>An "a=rtcp" line, as specified in
```

Comment:
This is a bit of a run-on sentence. I would try to work this into the para below by doing something like

    If present in the offer, an "a=rtcp-mux" line, as specified [...]. 
    Otherwise, an "a=rtcp", as specified [...], unless the "require" 
    RTCP multiplexing policy is set, in which case the m= line 
    MUST be marked as rejected [...]